README.md

```diff
@@ -382,3 +382,72 @@ Clone projects from the following links and run mvn clean install
 [WebSocket Message Streamer library](https://github.com/Engineering-Research-and-Development/market4.0-websocket_message_streamer)
 
 [Execution Core Container](https://github.com/Engineering-Research-and-Development/true-connector-execution_core_container)
+
+### GitHub Workflow
+
+This repository implements following branch management:
+
+![diagram](doc/workflow//github_actions_workflow.drawio.png?raw=true "GitHub Workflow diagram")
+
+and has several GitHub action files to support such functionality. Those files are located in:
+
+*.github\workflows\*  
```

Comment:
For some reason this does not render in bold, if that was you intent. Everything else is fine regarding readme and the rest of the PR.